src/pages/Calculator.js

```diff
@@ -0,0 +1,57 @@
+import { useState, React } from 'react';
+import '../styles.css';
+import calculate from '../logic/calculate';
+import Digit from '../components/Digit';
+import Operation from '../components/Operation';
+import Navbar from '../components/Navbar';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place.